### DISCUSSION TITLE
"error: No Container parent" means what?

### DISCUSSION THREAD

**Discussion by udippel** [gerbera/gerbera#2890] [General]
I have a - for me new - error popping up in the logs, no debug option set:
7 consecutive items directly in the parent directory of autoscan (Movies) with
`error: No Container parent for Item /../`
Strangely enough, 3 out of these 7 mention an identical file, that exists only once:
```
error: No Container parent for Item /.../Movies/1.000_Arten,_Regen_zu_beschreiben.mp4
error: No Container parent for Item /.../Movies/1.000_Arten,_Regen_zu_beschreiben.mp4
error: No Container parent for Item /.../Movies/1.000_Arten,_Regen_zu_beschreiben.mp4
```
The files can be played with VLC, directly on the machine as well as by clicking in the Database view.

Any idea, what produces this?

[Edit: the database was created from scratch here, with ...73..., method grb, a few days ago.]

**Comment by KarlStraussberger** [comment]
This is a programming error that should not have happend and is fixed from my perspective
